components/editor/modules/meta/ui.js

```diff
@@ -18,6 +18,7 @@ import AudioForm from './AudioForm'
 import UIForm from '../../UIForm'
 import DarkModeForm, { DARK_MODE_KEY } from './DarkModeForm'
 import ShareImageForm from './ShareImageForm'
+import GooglePreview from './GooglePreview'
```

Comment:
The other previews are all in the styleguide – I get that they have to do with ShareImage but it feels sad that this one's here all alone.